### ISSUE TITLE
Is it possible to make this project multi lingual?

### ISSUE THREAD

**Issue by animemandir** [AlexGarrixen/vimox-frontend#140]
Hola!!, senor Alex, The subtitles and all the visible text are in Spanish. Is it possible to make it multilingual?
I mean, the default language would be English as it is known worldwide, then at the top, there would be an option to select another language, and the anime episodes would also be based on the user choice.

**Comment by AlexGarrixen** [comment]
I have it in mind, so possibly implement it soon

**Comment by animemandir** [comment]
Yeah, that would be really awesome, or you can just teach how to do it? I can help ig.....I mean, you must have used Spanish api url.
Or is there any language value that fetches the data in Spanish, if you can show me how to change it..
